docs/components/console/manage-organization/alpha-terms.md

```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
```

Comment:
I recommend also changing the filename and ID for consistency. The goal isn't to talk about terms, but how to get cool new features